releng/org.eclipse.tracecompass.testing/feature.xml

```diff
@@ -207,4 +207,8 @@
          id="org.eclipse.tracecompass.slf4j.binding.simple.properties"
          version="0.0.0"/>
 
+   <plugin
+         id="org.eclipse.tracecompass.datastore.core.tests"
```

Comment:
Consider inserting above in alphabetical order

Reply:
The Eclipse feature editor added it at the end.